Blocks/MMQ/mmpq.h

```diff
@@ -74,8 +74,9 @@ class MMPQ {
   // NOTE(dkorolev): `std::enable_if_t<std::is_same<message_t, current::decay<T>>::value, idxts_t>` can't convert
   // `const char*` into an `std::string`, which is essential, as, unlike MMQ, MMPQ is not an `ss::EntryPublisher<>`.
   template <current::locks::MutexLockStatus MLS = current::locks::MutexLockStatus::NeedToLock, class T>
-  idxts_t Publish(T&& message, std::chrono::microseconds timestamp = current::time::Now()) {
+  idxts_t Publish(T&& message, std::chrono::microseconds us = std::chrono::microseconds(-1)) {
```

Comment:
Why not use `std::chrono:microseconds(-1)` as a constant, and compare to it directly, w/o `.count()`?